packages/cli/src/services/minecraft/minecraftUtility.ts

```diff
@@ -144,20 +154,33 @@ export class MinecraftUtility {
       const content = fs.readFileSync(blockModelsPath + filename, `utf-8`)
       const blockModelName = filename.split(`.`)[0]
       const rawBlock = JSON.parse(content) as BlockModelData
+
       let convertedBlock = {
         textures: {},
       } as BlockModelData
       if (!!rawBlock.textures) {
         Object.keys(rawBlock.textures).forEach((textureKey) => {
           if (!rawBlock.textures![textureKey]!.includes(`#`)) {
+            let texturePathFragment:
+              | string
+              | string[]
+              | undefined = rawBlock.textures![textureKey]
```

Comment:
:thinking: 

Que paso? Looks interesting

Reply:
I don't remember setting this up, but I did some research and it looks like it's a [type alias.](https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-aliases) Neat!